app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextModuleActions.java

```diff
@@ -306,4 +328,68 @@ public boolean onLongClick(View v) {
             return new ArrayList<>();
         }
     }
+
+    public class SttTaskComparator implements Comparator<SttTask> {
```

Comment:
Setting SttTaskComparator within SttCommander or SttTask throws an error since they are not enclosing classes. I've created a new file for SttTaskComparator next to SttTask.java instead of enclosing it within another class, since as you mentioned, it doesn't use any internal references to TextAction